cmds/ash/main.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2012-2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/u-root/u-root/pkg/complete"
+	"github.com/u-root/u-root/pkg/termios"
+)
+
+func main() {
+	t, err := termios.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	r, err := t.Raw()
+	defer t.Set(r)
+	for {
+		p, err := complete.NewPathCompleter()
+		if err != nil {
+			log.Fatal(err)
```

Comment:
This error is triggered when `$PATH` is empty. I cannot think of one, but there might be a valid use case for `$PATH` to be empty.

Reply:
it's now an error to have it be empty. Note that os.Getenv doesn't let you tell if the variable did not exist or was empty.